VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -71,6 +72,7 @@
                 android:clickable="false"
                 android:focusable="false"
                 android:focusableInTouchMode="false"
+                android:contentDescription="@string/go_to_video"
```

Comment:
This is on the course discovery screen. Some courses have an externally hosted video link (e.g. YouTube) associated with it. We display a play icon. This sets content description of that icon to be "Go to video"